.github/workflows/hardhat-test.yml

```diff
@@ -0,0 +1,48 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Hardhat test
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  hardhat-test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    env:
+      DEPAY_MNEMONIC: ${{secrets.DEPAY_MNEMONIC}}
+      DEPAY_RPC_URL: ${{secrets.DEPAY_RPC_URL}}
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Echo
```

Comment:
you can remove this, that was just for testing ;)

Reply:
It had been removed, please merge.